includes/admin/meta-boxes/views/html-variation-admin.php

```diff
@@ -48,6 +48,16 @@
 		?>
 		<input type="hidden" name="variable_post_id[<?php echo esc_attr( $loop ); ?>]" value="<?php echo esc_attr( $variation_id ); ?>" />
 		<input type="hidden" class="variation_menu_order" name="variation_menu_order[<?php echo esc_attr( $loop ); ?>]" value="<?php echo esc_attr( $variation_object->get_menu_order( 'edit' ) ); ?>" />
+		
+		<?php
+		/**
+		 * woocommerce_variation_header action.
+		 *
+		 * @since 3.6.0
+		 *
+		 * @param WP_Post $variation
+		 */
+		do_action( 'woocommerce_variation_header', $variation ); ?>
```

Comment:
Closing `?>` should probably go underneath `do_action()` to follow upstream coding standards.

Reply:
Resolved in a9afc72